feature1/one-more-file.txt

```diff
@@ -1 +1,3 @@
-This file was also added in the feature1 branch
\ No newline at end of file
+This file was also added in the feature1 branch
```

Comment:
It is good that you have decided to add this line of code,

Reply:
Yeah, I can see it. Thanks